.github/workflows/build-test-distribute.yaml

```diff
@@ -34,19 +34,14 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      # GitHub actions does not share cache across multiple jobs,
```

Comment:
shifting these snippets to make it more readable and align ordering with other jobs.